### PR TITLE
finiished quichsort with passed tests

### PR DESCRIPTION
**Pull request by xialiu1988**
- [ ] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: 401code_challenges/codeChallenge/src/main/java/quickSort/App.java [hunk 1]
```diff
@@ -2,6 +2,61 @@
...
+            if(left<right){
+                int temp=arr[left];
+                arr[left]=arr[right];
+                arr[right]=temp;
+                left++;
+            }
+
+        }
+
+return right;
```
Comment (evanslaton) [inline 401code_challenges/codeChallenge/src/main/java/quickSort/App.java]: improperly indented

File: 401code_challenges/codeChallenge/src/main/java/quickSort/App.java [hunk 2]
```diff
@@ -2,6 +2,61 @@
...
+        if(left>right) return ;
+        else {
+            int partition = sortit(arr, left, right);
+            quicksort(arr, left, partition - 1);
+            quicksort(arr, partition+1, right);
+        }
+    }
+
+    public static int sortit(int[] arr,int left,int right){
+
```
Comment (evanslaton) [inline 401code_challenges/codeChallenge/src/main/java/quickSort/App.java]: Lot's of extra white space
